reference/forms/types/options/choice_value.rst.inc

```diff
@@ -12,7 +12,10 @@ about this, but it might be handy when processing an API request (since you can
 configure the value that will be sent in the API request).
 
 This can be a callable or a property path. See `choice_label`_ for similar usage.
```

Comment:
Maybe it's better to remove ``"See `choice_label`_ for similar usage.`` completely. This would avoid talking about "differences" with the other options.